### ISSUE TITLE
Run tests on IE and Edge

### ISSUE THREAD

**Issue by jdanekrh** [jdanekrh/dispatch-console-tests#21]
- [x] Think about possibilities
- [x] Estimate effort and costs
- [x] Set something up and see how it works

**Comment by jdanekrh** [comment]
* AppVeyor: cannot run Dispatch itself; could run the fake data producing script
* AppVeyor and something else: problem with coordination, needs custom orchestration
* Internal RH QE infra: needs a way to publish results publicly, cannot just build random PRs, cannot reach it from outside
* some cloud provider: have to pay Windows licence, need to publish data myself, need to run some CI, has costs

**Comment by jdanekrh** [comment]
My chosen solution was SauceLabs and Google Compute Engine running the docker images. Before I got to implementing that, I looked once more at SauceLabs and the original WebSocket issue I was having, and found out http://stackoverflow.com/questions/24381538/websocket-on-saucelabs-immediately-drops-connection-code-1006/43557765.

This is best solved by just using SauceLabs with the no_bump_ssl option as described above.
